doc/run_on_kubernetes.md

```diff
@@ -0,0 +1,59 @@
+# Running SQLFlow on Kubernetes
+
+This is a tutorial on how to run SQLFlow on Kubernetes, and this tutorial will deploy:
+- MySQL server with the SQLFlow example datasets loaded, and
+- SQLFlow components contain the sqlflow-server the Jupyter notebook.
+
+Then you can run the SQLFlow query in the Jupyter notebook on your web browser.
+
+## Preparation
+
+1. Setup a Kubernetes cluster: You can refer to the [official page](https://kubernetes.io/docs/setup) to set up a 
+full cluster or use a local quick start tool: [minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/)
+This tutorial would use [minikube] to demonstrate the SQLFlow.
+1. [Install the kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/), which is the command line tool
+to interact with the Kubernetes cluster.
+1. Pull the official Docker image [sqlflow/sqlflow:latest](https://hub.docker.com/r/sqlflow/sqlflow) or [build
+from source](/doc/build.md)
+
+## Deploy the SQLFlow Components
+
+1. Deploy the MySQL service and sqlflow service in the difference containers in one Pod. 
+    ``` bash
+    > kubectl create -f k8s/sqlflow-mysql.yaml
+    ```
```

Comment:
We need an explanation on what the above command does, like

> The above command starts a pod with two containers, both running the `sqlflow/sqlflow:latest` Docker image, but on container runs the MySQL server instance and the other one runs the SQLFlow server instance.

I am not sure if my above suggested edit matches the sqlflow-mysql.yaml, but I am curious as it seems not starting the Jupyter Notebook server?

Reply:
Thanks, will add some comment to explain the k8s command.
Currently, this PR would start the Jupyter Notebook server and sqlflow-server in the same container, but as your [comment](https://github.com/sql-machine-learning/sqlflow/pull/475#discussion_r289538186), I think to separate the two process in the different container is a better way.